README.md

```diff
@@ -21,3 +21,26 @@ Presentation
 Report
 ---
 #### 
+
+# Documentation
+
+Getting started - Linux
+---
+####
+These steps will work for any Linux distribution with systemd.
+1. Install Docker Desktop, Earthly, and jupyter lab.
```

Comment:
* Don't need jupyter lab installed as of #9 because it's included in the docker image
* No longer need earthly installed as of #10

Reply:
Updated in #15